inst/gitlab-ci/check-coverage-pkgdown.yml

```diff
@@ -29,126 +29,127 @@ building:
     - Rscript -e 'remotes::install_local(upgrade = "never")'
     - R -e 'rcmdcheck::rcmdcheck(args = c("--no-manual", "--as-cran"), error_on = "warning", check_dir = "check")'
   artifacts:
-      paths:
-          - check
-      expire_in: 30 days
-      
+    paths:
+      - check
+    expire_in: 30 days
+
 # To have the coverage percentage appear as a gitlab badge follow these
```

Comment:
Can you add an information here to specify that from version > 15.0, this is not anymore necessary.

Reply:
Done